hoptimator-k8s/src/main/java/com/linkedin/hoptimator/k8s/K8sEngineTable.java

```diff
@@ -72,12 +72,7 @@ private static SqlDialect dialect(Row row) {
     if (row.DIALECT == null) {
       return SqlDialect.ANSI;
     }
-    switch (row.DIALECT) {
-    case "Flink":
-      return SqlDialect.FLINK;
```

Comment:
Think we still need this. If an engine has  `dialect: Flink` that needs to map to FLINK enum here.

Reply:
Note there are two concepts of dialect at play. Calcite has a sql dialect but we layer on our own. Our `SqlDialect.FLINK` should basically be Calcite's ANSI plus whatever Flink-specific DDL we need to generate.